test/runexamples.jl

```diff
@@ -0,0 +1,15 @@
+using Test
+
+if run_examples
+    @testset "Run examples" begin
+        basedir = joinpath(@__DIR__, "..", "examples")
+        for file in readdir(basedir)
+            if isfile(joinpath(basedir, file)) && endswith(file, ".jl")
+                file == "graphene.jl" && continue  # Has to be run on cluster
```

Comment:
can you take that out of the loop as a list of exceptions?

Reply:
I'll solve that by adding markers inside the file. I think that's better (info is locally inside the example file).